src/api/parity_set.rs

```diff
@@ -0,0 +1,240 @@
+use api::Namespace;
+use helpers::{self, CallFuture};
+use types::{H256, Address};
+
+
+use Transport;
+
+#[derive(Debug, Clone)]
+/// `Parity_Set` Specific API
+pub struct ParitySet<T> {
+    transport: T
+}
+
+impl<T: Transport> Namespace<T> for ParitySet<T> {
+    fn new(transport: T) -> Self
+        where
+        Self: Sized,
+    {
+        ParitySet { transport }
+    }
+
+    fn transport(&self) -> &T {
+        &self.transport
+    }
+}
+
+impl<T: Transport> ParitySet<T> {
+    
+    /// Set Parity to accept non-reserved peers (default behavior)
+    pub fn parity_accept_non_reserved_peers(&self) -> CallFuture<bool, T::Out> {
```

Comment:
Can we drop the `parity_` prefix?

Reply:
Yes! With `parity_` prefix its a bit repetitive :)